src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/ProcessCompletionOfNextDataSetVersionFunction.cs

```diff
@@ -3,6 +3,7 @@
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
 using GovUk.Education.ExploreEducationStatistics.Public.Data.Services.Interfaces;
+using LinqToDB;
```

Comment:
Whilst this works, probably should be using `SingleAsync` from `Microsoft.EntityFrameworkCore`

Reply:
Aha good spot, updated thanks.